hw/top_earlgrey/data/top_earlgrey_rnd_cnst_pkg.sv.tpl

```diff
@@ -0,0 +1,43 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+${gencmd}
+<%
+  def make_blocked_sv_literal(hexstr, randwidth):
```

Comment:
does it make any sense to move this to `topgen/lib.py`?
Can't quite tell if others would ever make use of it...

Reply:
not sure... it feels pretty specific to this use case, since I think it is currently the only place where we have automatically generated hexstrings with more than 100 chars.